webapp/packages/plugin-connections/src/ConnectionForm/Options/ConnectionOptionsTabService.ts

```diff
@@ -189,6 +216,7 @@ export class ConnectionOptionsTabService extends Bootstrap {
     }
 
     if (!state.info) {
+      await this.setDefaults(state);
       return;
     }
```

Comment:
it seems like it can reset the form on 'test connection' (when we creating new connection)